nullable_types.go

```diff
@@ -0,0 +1,166 @@
+package graphql
```

Comment:
Added Support for nullable types https://github.com/tokopedia/graphql-go/commit/fced4f616be9057a68c29119bec96ed2106271cf